l10n_eu_product_adr_local/models/adr_goods.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, fields
+
+class AdrGoods(models.Model):
+    _inherit = 'adr.goods'
+
+    un_local_code = fields.Char(required=True)
```

Comment:
This is right, but... we should consider how to manage initial installation scenario... we'll talk about it